themes/classic/templates/catalog/_partials/product-variants.tpl

```diff
@@ -35,4 +35,5 @@
       {/if}
     </div>
   {/foreach}
+  <input type="hidden" name="idCombination" id="idCombination" value="{$product.id_product_attribute}">
```

Comment:
P.S. I am talking about ability to know combination ID after ajax-call. Many modules uses idCombination value for determine active/current combination at the Product page. 

P.P.S. link to the default-bootstrap theme: https://github.com/PrestaShop/PrestaShop/blob/1.6.1.x/themes/default-bootstrap/product.tpl#L254